SCClassLibrary/Common/Control/Spec.sc

```diff
@@ -100,7 +100,15 @@ ControlSpec : Spec {
 		^numStep
 	}
 
-	grid { ^grid ?? {GridLines(this)} }
+	grid {
+		^grid ?? {
+			warp.switch(
+				LinearWarp(), {GridLines(this)},
+				ExponentialWarp(), {ExponentialGridLines(this)},
+				{GridLines(this)} // all other Warps
```

Comment:
style: spaces padding enclosing curly brackets (style is mixed in this doc, so I think new  code may as well follow the 'rules'), e.g.
```supercollider
LinearWarp(), { GridLines(this) },
```